### ISSUE TITLE
Multiplication issue

### ISSUE THREAD

**Issue by raedon707** [nefe/number-precision#41]
I'm trying to multiply 9.700000042 and 1000000000 using the `times` method.
It should show 9700000042 as the final result. But it's returning 9700000041.999998.
please help.

**Comment by nmorse** [comment]
yes strange and if you add or remove a zero from each number it comes out as expected:
9.70000042 * 100000000 -> as expected
9.700000042 * 1000000000  -> NOT as expected
9.7000000042 * 10000000000 -> as expected
